plugins/dynamodb/komand_dynamodb/actions/scan/action.py

```diff
@@ -39,11 +39,4 @@ def run(self, params={}):
             kwargs["IndexName"] = index
 
         results = t.scan(**kwargs)
-        return {
```

Comment:
If the return is being removed, how are the results from this action being used?

Reply:
@pmara-r7 the return isn't being removed - there is just a different output being returned now.